civihr_employee_portal/js/reports.js

```diff
@@ -919,6 +919,67 @@
                         }
 
                         this.$DOM.sections.dataWrapper.fadeIn();
+                        var $DOM = this.$DOM;
+
+                        $.ajax({
```

Comment:
Apologies, what I wrote above was still for a sequential approach, obviously the parallel approach would be something like:
```js
// data table
$.ajax({
}).then(function (data) {
}

// summary table
$.ajax({
}).then(function (data) {
}

```